pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -89,6 +89,17 @@ const (
 	// StateDeleting means we are in the process of cleaning up the host
 	// ready for deletion
 	StateDeleting ProvisioningState = "deleting"
+
+	// StateSoftPowerOffTriggered means host if soft powered off
+	// but not yet finished
+	StateSoftPowerOffTriggered ProvisioningState = "soft power off triggered"
+
+	// StateSoftPowerOffUnsupported means soft powered off command
+	// is not supported by the driver
+	StateSoftPowerOffUnsupported ProvisioningState = "soft power off unsupported"
+
+	// StateSoftPowerOffError means soft powered off fails
+	StateSoftPowerOffError ProvisioningState = "soft power off error"
```

Comment:
We don't have a separate hard power off error. Do we need to track errors with soft power off?